notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>Could you add a bit about the context why G-R and R-I were used here? What they represent?</p><p> </p><ul><li>A typo: CMd --&gt; CMD</li></ul><p> </p><ul><li>Please write the magnitudes in lower case. G --&gt; g, G-R --&gt; g-r, R-I --&gt; r-i and other upper case letters to the lower case in the bullets.</li><li>Please refer <code>color</code> to</li></ul><pre class="ql-syntax" spellcheck="false">the difference between the magnitudes in that filter and the magnitudes in another filter
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>